eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -129,4 +144,16 @@ public void exec() throws IOException {
         }
     }
 
+    /**
+     * Construct severities to fail on.
+     * @param failonwarn Fail on warning flag
+     * @return Constructed set
+     */
+    private static Set<ErrorSeverity> failOnSeverities(final boolean failonwarn) {
```

Comment:
@mximp composite names are code smell: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html How about just `fail`?

Reply:
Sometimes it's tricky to follow this one.